peer.go

```diff
@@ -61,15 +60,21 @@ func Default(options PeerOptions, receiver MessageReceiver, sender MessageSender
 		panic(fmt.Errorf("pre-condition violation: %v", newErrInvalidPeerOptions(err)))
 	}
 
-	if options.DHTStore == nil {
-		options.DHTStore = kv.NewMemDB()
+	if options.Store == nil {
+		options.Store = kv.NewMemDB()
 	}
 
-	if options.BroadcasterStore == nil {
-		options.BroadcasterStore = kv.NewMemDB()
+	dhtTable, err := options.Store.NewTable("DHT", kv.GobCodec)
+	if err != nil {
+		panic(fmt.Errorf("failed to initialize DHT table: %v", err))
+	}
+
+	broadcasterTable, err := options.Store.NewTable("Broadcaster", kv.GobCodec)
```

Comment:
Don’t name tables based on who is using them. Name them based on the data inside. For example, instead of `“DHT”` use `”peerAddrs”`. Instead of `”Broadcaster”` use `”broadcastMessageHashes”`.